Remotion/ObjectBinding/Web.Test/IndividualControlTests/BocListUserControl.ascx.cs

```diff
@@ -354,6 +353,17 @@ override protected void OnLoad (EventArgs e)
   protected override void OnPreRender (EventArgs e)
   {
     base.OnPreRender(e);
+
+    var haveValidationErrors = (bool?)Session[c_haveValidationErrors] ?? false;
+    ChildrenHaveValidationErrorsCheckbox.Checked = haveValidationErrors;
+    if (haveValidationErrors)
```

Comment:
I think there should be a line break before the condition, check other usages too :)
```suggestion
    
    if (haveValidationErrors)
```